bedrock/firefox/redirects.py

```diff
@@ -594,6 +594,8 @@ def mobile_app(request, *args, **kwargs):
     redirect(r"^firefox/flashback/?$", "firefox"),
     # issue 14222
     redirect(r"^firefox/browsers/?$", "firefox"),
+    # no issue
+    redirect(r"^firefox/privacy/?$", "privacy"),
     # issue 14248
     redirect(r"^firefox/privacy/products/?$", "products.landing"),
```

Comment:
also here it can be grouped as it's the same issue#:

```suggestion
    # issue 14248
    redirect(r"^firefox/privacy/?$", "privacy"),
    redirect(r"^firefox/privacy/products/?$", "products.landing"),
```